script/Ciel/Day 13/13_01_01_CIEL13_1.txt

```diff
@@ -725,12 +779,12 @@ The final memories of an unfamiliar someone――someone who secluded themselves
 [sha:081e061b6766e7f8b0153cac61a93affb27c4848]{
 -- Page 38, Offset 33210.
 -- ……手段がない。
-...There's no solution.
+There's no way out...
 }
 [sha:db482298044c8dfa8928f6641fbbc40793fc895e]{
 -- Page 38, Offset 33211.
 -- コイツは、俺が自殺すれば解決するという問題でもない。
-This is not something that can be solved by killing myself.
+This isn't something I can solve by killing myself.
 }
 [sha:33a127bdda299095de953aceae799c6b3c95fa35]{
 -- Page 38, Offset 33212.
```

Comment:
This one seems a bit misinterpreted, but it's a hard thing to translate regardless.
>And there's nothing I can do to change that.

Reply:
Yeah this one is annoying. I think the current ver works fine as a sentence that makes sense in context, but if it's too focused on the extrinsics maybe you could do sth like
> I can't escape the reality of my situation.

"There's nothing I can do to change the fact that I can't solve this by killing myself" just seems so odd as a complete thought

Reply:
I think it goes like
……手段がない。 (nothing can do become)
コイツは、俺が自殺すれば解決するという問題でもない。(because... killing myself won't help)
こればかりは、もうどうしようもない。(so there truly is nothing about that which can be done!)

and I suppose it's more about finding the phrasing which fits while still letting this ないx3 ending feel like its properly incorporated.